HyperVExtension/src/HyperVExtension/Providers/HyperVProvider.cs

```diff
@@ -75,9 +84,8 @@ public IAsyncOperation<ComputeSystemsResult> GetComputeSystemsAsync(IDeveloperId
 
     public ComputeSystemAdaptiveCardResult CreateAdaptiveCardSessionForDeveloperId(IDeveloperId developerId, ComputeSystemAdaptiveCardKind sessionKind)
     {
-        // This won't be supported until creation is supported.
-        var notImplementedException = new NotImplementedException($"Method not implemented by Hyper-V Compute System Provider");
-        return new ComputeSystemAdaptiveCardResult(notImplementedException, OperationErrorString, notImplementedException.Message);
+        var imageList = _vmGalleryService.GetGalleryImagesAsync().GetAwaiter().GetResult();

```

Comment:
Can we not await this?

Reply:
I'd have to change the SDK idl interface to support an IAsyncOperation. At some point somewhere in this method there needs to be a synchronous wait as we need to send the operation back to Dev Home in the return call since this call isnt awaitable.